ADBench/run-all.ps1

```diff
@@ -451,8 +462,9 @@ $tool_descriptors = @(
     [Tool]::new("ManualEigenVector", "cpp", [ObjectiveType] "GMM", $true, $default_tolerance)
     [Tool]::new("Tapenade", "cpp", [ObjectiveType] "BA, LSTM, GMM, Hand", $true, $default_tolerance)
     [Tool]::new("DiffSharp", "bin", [ObjectiveType] "BA", $false, 0.0, $true, $false)
+    [Tool]::new("PyTorch", "python", [ObjectiveType] "BA, LSTM, GMM, Hand", $true, $default_tolerance)
     [Tool]::new("Autograd", "py", [ObjectiveType] "GMM, BA", $false, 0.0, $true, $false)
-    [Tool]::new("PyTorch", "py", [ObjectiveType] "GMM, LSTM", $false, 0.0)
+    #[Tool]::new("PyTorch", "py", [ObjectiveType] "GMM, LSTM", $false, 0.0)
```

Comment:
Can't we just delete the old pytorch benchmark, or is there something we still need to preserve?

Reply:
We can delete it if it's always slower -- if it's faster, we need to beat it before we delete it.

Reply:
@mikhailnikolaev can we plot graphs for the old pytorch benchmark alongside the new one?

Reply:
Ok, I will plot such graphs and will check efficiency

Reply:
@awf and @msdmkats, I have created the graphs, they are attached to the PR description now, replacing the old version of graphs. As you can see, in the new version jacobians are calculated faster, but objectives, ont the other hand, by some reason - slower